web/setupLogger.ts

```diff
@@ -1,13 +1,26 @@
 import { isCommandArg } from '../common/util/basic';
-import { configure } from 'log4js';
+import { configure, addLayout } from 'log4js';
+
+addLayout('json', function () {
+    return function (logEvent) {
+        const message = logEvent.data.shift();
+        return JSON.stringify({ ...logEvent, message });
+    };
+});
+
+let appenders = ['outJson'];
+if (process.env.ENV === 'dev') {
```

Comment:
We use ENV = 'dev' for review apps & staging. Maybe we should do something like ENV != local or add a different env to turn this on?

Reply:
Ah, good to know. I'll add a different env.